src/components/Icon/Icon.tsx

```diff
@@ -11,9 +11,8 @@ type StyleProps = {
 const useStyles = makeStyles(() => ({
   icon: {
     '& path': {
-      fill: (props: StyleProps) => props.fillColor ?? 'inherit',
-    }
-  }
+      fill: (props: StyleProps) => props.fillColor ? props.fillColor : 'initial',
```

Comment:
@nickgraz the other change didn't seem to work when applying the fill color, trying this now